components/Article/ArticleGallery.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
```

Comment:
Just for consistency: We typically do
`import React, { Fragment } from 'react'`
so we don't have to namespace the component further down.